clusters/build-clusters/common_managed/README.md

```diff
@@ -0,0 +1,3 @@
+This directory is used to store common objects we used only in managed clusters (e.g., build01-10).
```

Comment:
Could you also give an example of the negative case?
Would a hypershift cluster counted as managed? I know there isnt one today. Could be one tomorrow.

Reply:
Are the managed clusters defined here too?
https://github.com/openshift/release/blob/master/clusters/build-clusters/_cluster-init.yaml

Reply:
Hypershift clusters are just type of clusters similar as OSD vs OCP.
Yes.

Reply:
Could a hypershift cluster be considered managed?

Reply:
This one is still unaddressed.
> Could you also give an example of the negative case?

Reply:
why we give negative case there? I think is just make things more confusing.

Reply:
> Could a hypershift cluster be considered managed?
Same as asking `could a OSD cluster be considered managed`? Its just depends on how you configure the cluster.

Reply:
Let me rephrase:
I would like to know if this folder should be applied to a given cluster.
At the moment, we use this algorithm to determine: OCP, OKD, OSD.

```
oc get svc -n openshift-monitoring configure-alertmanager-operator
```
If the above `svc` exists, then OSD.
Otherwise, 

```console
version=$(oc get clusterversion version --context build01 -o yaml| yq '.status.history[0].version')
```
If "$version" contains 'okd', then the cluster is OKD, otherwise, OSD.

So ... now we have a new type "managed" and another one "hosted". How do we extend the above algorithm to determine if or not a cluster belongs to the type?